BlackNectar/BlackNectar/StoresMapViewController.swift

```diff
@@ -80,29 +91,33 @@ class StoresMapViewController: UIViewController, MKMapViewDelegate, CLLocationMa
         
         self.mapView.setRegion(region, animated: true)
     }
-
-    // populating stores as annotations in the mapView
+  
+    // populating stores as annotations in the mapView    
     func populateStoreAnnotations() {
-      
-        for store in stores {
-
-            if store != nil {
-
-                let storeName = store.storeName
-                let address = store.address.allValues
-                let location = store.location
-
-                let latitude = location.latitude
-                let longitude = location.longitude
+        
+        if stores != nil {
```

Comment:
The Swift way of handling optionals is to use Optional Unwrapping, and not manually checking for nil.

```swift
if let store = store {
    ...
}
```